docker/main/Dockerfile.base

```diff
@@ -0,0 +1,25 @@
+# Dockerfile.base
+FROM pytorch/pytorch:latest
+
+# Install tzdata / git
+RUN apt-get update && \
+    ln -fs /usr/share/zoneinfo/America/New_York /etc/localtime && \
+    apt-get install -y tzdata && \
+    dpkg-reconfigure --frontend noninteractive tzdata && \
+    apt-get -y install --no-install-recommends git && \
+    rm -rf /var/lib/apt/lists/*
+
+# Ignite main dependencies
+RUN pip install --upgrade --no-cache-dir pytorch-ignite \
+                                         tensorboard \
+                                         tqdm
+# Checkout Ignite examples only
+RUN mkdir -p pytorch-ignite-examples && \
```

Comment:
@fco-dv I'm not sure to understand what the following does and if we keep only examples instead of complete repository ?

Reply:
Yes it's like a checkout filtered on the folder "examples" ( I've just learnt that it was possible) , I have checked that the whole repo was not pulled in a running container. However there may be a more elegant way to do this sparse checkout.